srcs/main.c

```diff
@@ -6,7 +6,7 @@
 /*   By: ldominiq <ldominiq@student.42.fr>          +#+  +:+       +#+        */
 /*                                                +#+#+#+#+#+   +#+           */
 /*   Created: 2022/04/07 16:09:46 by tgrivel           #+#    #+#             */
-/*   Updated: 2022/06/24 15:03:35 by tgrivel          ###   ########.fr       */
+/*   Updated: 2022/06/25 20:41:54 by lgyger           ###   ########.fr       */
```

Comment:
Header changed but no change inside...
```suggestion
/*   Updated: 2022/06/24 15:03:35 by tgrivel          ###   ########.fr       */
```